srcpkgs/editorconfig-core-lua/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'editorconfig-lua'
+pkgname=editorconfig-core-lua
+version=0.2.0
+revision=1
+build_style=cmake
+hostmakedepends=""
+makedepends="editorconfig-devel lua-devel"
+depends="lua"
```

Comment:
lua is already found by shlibs

```
=> editorconfig-core-lua-0.2.0_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libeditorconfig.so.0 <-> editorconfig>=0.12.2_1
   SONAME: liblua5.3.so.5.3 <-> lua>=5.3.1_2
   SONAME: libc.so.6 <-> glibc>=2.28_1
```

Reply:
Cool, didn't know about this automatic detection, BRB

Reply:
Yes, it is documented on https://github.com/void-linux/void-packages/blob/master/Manual.md#runtime-dependencies